### ISSUE TITLE
Tests for XML testing of Responses 

### ISSUE THREAD

**Issue by In-vincible** [fossasia/searss#32]
We need to add tests for testing the received responses, I wish to take on this issue @rhemon @jayanthchandra , I have added the issue, can I get the abstract of the tests to be performed, like first thing we will check if the xml received is valid, I don't think we can just add static data for testing exact results as the results may change with time, please help :)

**Comment by rhemon** [comment]
yeah you have to use regexp I guess, with which you can test whether xml valid as well as confirm its giving data.

**Comment by In-vincible** [comment]
https://github.com/Exirel/python-xmlunittest/blob/master/doc/index.rst I think we should use this or any other library for testing XML, should I proceed with this one @rhemon

**Comment by rhemon** [comment]
yeah i think you can use that and proceed, later on PR if thing seems wrong
we can change according to me. and also using that will make tests easier i
think.
